libvast/vast/system/node.hpp

```diff
@@ -44,7 +59,7 @@ struct node_state {
   tracker_type tracker;
 
   /// Stores how many components per label are active.
-  std::unordered_map<std::string, size_t> labels;
+  std::map<std::string, size_t> labels;
```

Comment:
Why this change? Seems unrelated to the PR...

Reply:
That's true. I don't remember why I changed it. Probably because this map is small and the hash map doesn't pay off for small maps. Would you like me to change it back?

Reply:
I don't think it makes a difference either way. I asked out of curiosity. Let's keep it and merge.

Reply:
@mavam then it's up to you to merge or add another review round. :)